app/routes/notices-setup.routes.js

```diff
@@ -233,6 +233,18 @@ const routes = [
       }
     }
   },
+  {
+    method: 'GET',
+    path: basePath + `/{sessionId}/check-notice-type`,
```

Comment:
can we remove the basePath as part of this PR? to standardise it with the rest of the controllers?